src/Tribe/Views/V2/View.php

```diff
@@ -2243,4 +2244,17 @@ protected function setup_additional_views( array $events = [], array $template_v
 			$latest_past_view->add_view_filters();
 		}
 	}
+
+	/**
+	 * Returns the number of upcoming events in relation to the "now" time.
+	 *
+	 * @since TBD
+	 *
+	 * @return int The number of upcoming events from "now".
+	 */
+	protected function upcoming_events_count() {
+		$from_date = tribe_beginning_of_day( $this->context->get( 'now', date( 'Y-m-d H:i:s' ) ) );
```

Comment:
:no_entry_sign: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.

<sub>hash: f15f34ccaf99715398a2576ff0d1f424</sub>